src/components/views/elements/PersistedElement.js

```diff
@@ -154,11 +159,20 @@ export default class PersistedElement extends React.Component {
     }
 
     renderApp() {
-        const content = <MatrixClientContext.Provider value={MatrixClientPeg.get()}>
-            <div ref={this.collectChild} style={this.props.style}>
-                { this.props.children }
-            </div>
-        </MatrixClientContext.Provider>;
+        const content = (
+            <MatrixClientContext.Provider value={MatrixClientPeg.get()}>
+                <div class={this.props.draggable ? "mx_CallContainer" : ''}>
```

Comment:
```suggestion
                <div className={this.props.draggable ? "mx_CallContainer" : ''}>
```

Reply:
I definitely need to figure out why eslint in vscode doesn't work for js here, this should've been caught.

Reply:
This isn't a thing ESLint checks for, my IDE does it as part of React support

![image](https://user-images.githubusercontent.com/2403652/126993801-90df73ea-bb94-49f5-aa7e-434474e78ed1.png)

Reply:
@t3chguy I'll take care of it then.